File: Problem (4): Median of Two Sorted Arrays .java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
         int[] merged = new int[nums1.length + nums2.length];
        System.arraycopy(nums1, 0, merged, 0, nums1.length);
        System.arraycopy(nums2, 0, merged, nums1.length, nums2.length);
        Arrays.sort(merged);
        int n = merged.length;
        if (n % 2 == 0) {
            int mid1 = merged[(n / 2) - 1];
            int mid2 = merged[n / 2];
            return (mid1 + mid2) / 2.0;
        } else {
            return merged[n / 2];
        }
    }
        
    }

File: Problem (4) : Median of Two Sorted Arrays.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        l=sorted(nums1+nums2)
        n=len(l)
        if (n%2==0):
            mid1=l[(n//2)-1]
            mid2=l[n//2]
            return float((mid1+mid2)/2.0)
        else:
            return float(l[(n/2)])
           
